docs/pages/reference/cli.mdx

```diff
@@ -12,6 +12,8 @@ Teleport is made up of four CLI tools.
 
 (!docs/pages/includes/permission-warning.mdx!)
 
+(!docs/pages/includes/cli-completion.mdx!)
```

Comment:
If the partial is only included in one place, should we just inline the content directly here?